calico_node/tests/st/policy/test_felix_gateway.py

```diff
@@ -140,216 +154,285 @@ def tearDownClass(cls):
         for host in cls.hosts:
             host.cleanup()
             del host
+        cls.calinet.delete()
 
         log_and_run("docker rm -f cali-st-ext-nginx || true")
 
-    def test_ingress_policy_can_block_through_traffic(self):
-        self.add_policy({
-            'apiVersion': 'v1',
-            'kind': 'policy',
-            'metadata': {'name': 'port80-int'},
-            'spec': {
-                'order': 10,
-                'ingress': [
-                    {
-                        'protocol': 'tcp',
-                        'destination': {'ports': [80]},
-                        'action': 'deny'
-                    },
-                ],
-                'egress': [
-                    {'action': 'deny'},
-                ],
-                'selector': 'role == "gateway-int"'
-            }
-        })
-        self.add_gateway_internal_iface()
-        retry_until_success(self.assert_host_can_not_curl_ext, 3)
+    def test_can_connect_by_default(self):
+        retry_until_success(self.assert_host_can_curl_local, 3)
+        retry_until_success(self.assert_gateway_can_curl_ext, 3)
+        retry_until_success(self.assert_host_can_curl_ext, 3)
+        retry_until_success(self.assert_hostwl_can_access_workload, 3)
+        retry_until_success(self.assert_workload_can_curl_ext, 3)
+
+        self.add_host_iface()
 
-    def test_ingress_policy_can_allow_through_traffic(self):
+        # Adding the host endpoints should break connectivity until we add policy back in.
```

Comment:
Should we test for that at this point?  All 5 checks? 
(local access ingress, local access egress, host-ext access, workload access ingress, workload access egress)

Otherwise what you're doing in this function is: 
- check all connectivity works, 
- add an interface, 
- add a policy
- check one of your connectivity checks still works. 

What if the (adding interface + add a policy) didn't have any effect?